test/data/vessel-valid/polar-data.json

```diff
@@ -0,0 +1,37 @@
+{
+  "uuid": "urn:mrn:signalk:uuid:c0d79334-4e25-4245-8892-54e8ccc8021d",
+  "performance": {
+    "polars": [
+      {
+        "91c8d96d-4bbd-4c2f-ab36-295b04e83589": {
+          "name": "dynamicPolar",
+          "$description": "Dynamic polar diagram from actual sailing",
```

Comment:
`$` prefix looks like a typo - or why should this field name be start with $ ?

Reply:
fixed